### PR TITLE
ContactForm Refactor

### PR DESCRIPTION
**Pull request by Tishasoumya-02**
#4460

### CODE REVIEW

**Review by sneridagh — APPROVED** [review]
LGTM!

/cc @nileshgulia1 @JeffersonBledsoe

**Review by nileshgulia1 — APPROVED** [review]

**Comment by Tishasoumya-02** [comment]
Sorry ,I caught this a bit late 🥲  ,I've added usePrevious() hook as the loaded and loading values wouldn't be the same at the same time,success toast will won't probably get rendered.

File: src/components/theme/ContactForm/ContactForm.jsx [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
+import { defineMessages, useIntl } from 'react-intl';
+import { Link, useHistory, useLocation } from 'react-router-dom';
 import { toast } from 'react-toastify';
 
-import { Form, Toolbar, Toast } from '@plone/volto/components';
 import { emailNotification } from '@plone/volto/actions';
-import { getBaseUrl } from '@plone/volto/helpers';
+import { useDispatch, useSelector } from 'react-redux';
+import { Form, Toolbar, Toast } from '@plone/volto/components';
+import { getBaseUrl, Helmet, usePrevious } from '@plone/volto/helpers';
```
Comment (nileshgulia1) [inline src/components/theme/ContactForm/ContactForm.jsx:11]: why some hooks are under helpers? 🤔

Comment (Tishasoumya-02) [reply src/components/theme/ContactForm/ContactForm.jsx:11]: not sure , but usePrevious was already there

**Review by nileshgulia1 — APPROVED** [review]

**Review by sneridagh — APPROVED** [review]

**Pull request merged**